packages/hash/frontend/src/components/GlideGlid/glide-grid.tsx

```diff
@@ -38,6 +51,23 @@ const GlideGrid: ForwardRefRenderFunction<DataEditorRef, DataEditorProps> = (
     [palette],
   );
 
+  const onColumnResize = useCallback(
```

Comment:
This conflicts with https://github.com/hashintel/hash/pull/1398 – we could potentially remove the logic from this PR and comment on resizing over there (since it'll make this one smaller)

Reply:
Removed this work from this PR in e30bfadbe51f8fc1333f96b6c88ed38d28591f4a in favour of #1398.